script/e2e-test.sh

```diff
@@ -309,9 +309,11 @@ installChartMuseum "${CHARTMUSEUM_VERSION}"
 pushChart apache 8.6.2 admin password
 pushChart apache 8.6.3 admin password
 
-# Setting up local Docker registry
-setupLocalDockerRegistry
-pushLocalChart
+# Setting up local Docker registry if not in GKE
+if [[ -z "${GKE_BRANCH-}" ]]; then
+  setupLocalDockerRegistry
+  pushLocalChart
+fi
```

Comment:
The local docker registry doesn't work right now on GKE as the current configuration requires the k8s cert to be copied for issuing the certs. The quickest solution: ignore those tests when in GKE.